registry/handlers/mail.go

```diff
@@ -0,0 +1,40 @@
+package handlers
+
+import (
+	"fmt"
+	"net/smtp"
+	"strings"
+)
+
+// Mail provides fields of email configuration for sending.
+type Mail struct {
+	Host, Port, Login, Password, FromAddr string
+	Insecure                              bool
+	ToAddr                                []string
+}
+
+// MailSending allows users to send email, only if mail parameters is configured correctly.
+func MailSending(mail Mail, subject, message string) error {
+	msg := fmt.Sprintf("To: %s\r\nFrom: %s\r\nSubject: %s\r\nContent-Type: text/html\r\n\r\n%s",
```

Comment:
Unless the body of the email (in this case a go panic) is rendered as html, this should be text/plain.